src/components/Spinner/SpinnerLoader.tsx

```diff
@@ -1,21 +1,34 @@
 import React from 'react';
 import {ActivityIndicator} from 'react-native';
+import {TextComponent} from 'src/styled-components/globalStyles/GlobalStyles.styled';
+import {defaultTheme} from 'src/styled-components/theme/theme.styled';
 import styled from 'styled-components/native';
 
 interface SpinnerProps {
-  color: string;
+  color?: string;
+  text?: string;
 }
 
-export const SpinnerLoader = ({color}: SpinnerProps): JSX.Element => {
+export const SpinnerLoader = ({
+  color = defaultTheme.colors.blueGray,
+  text = '',
+}: SpinnerProps): JSX.Element => {
   return (
     <WrapSpinnerLoad>
+      {text && (
+        <TextComponent fontSizeType="text" fontWeight="bold" color="darkBlue">
+          {text}{' '}
```

Comment:
delete {' '}

Reply:
this space separate the text from icon (component ActivityIndicator). Example: Cargando... (spinner), without this {' '} -> Cargando...(spiner) ... 
But I will delete it if you want.

Reply:
Okey. I wanted avoid it, but I agree